libs/blocks/modal/modal.css

```diff
@@ -179,3 +179,10 @@
     max-height: 100%;
   }
 }
+
+@media (min-width: 1200px) {
+  .dialog-modal.commerce-frame {
+    width: 1024px;
+    max-height: 825px;
+  }
+}
```

Comment:
This is adding a scroll in the commerce modal in desktop view which is not how it is on live right now. I have shared the details in the comment of this PR. Could you please check. Thanks.